app/services/zendesk_service.rb

```diff
@@ -0,0 +1,45 @@
+class ZendeskService
+  attr_accessor :ticket
+
+  class << self
+    def send!(ticket)
+      ZendeskService.new(ticket).send!
+    end
+  end
+
+  def initialize(ticket)
+    @ticket = ticket
+  end
+
+  def send!
+    ZendeskAPI::Request.create!(
+      client,
+      requester: { email: ticket['email_address'], name: ticket['full_name'] },
+      subject: ticket['subject'],
+      comment: {
+        body: ticket['message'],
+      },
+      custom_fields: [
+        { id: '360011490478', value: 'contact_form' },
+        { id: '360011798678', value: ticket['user_type'] },
+        { id: '360011519218', value: ticket['support_topics'] },
+        { id: '360011762698', value: ticket['telephone_number'] },
```

Comment:
I guess these IDs have been created in the Zendesk user interface? I'm a little uncomfortable having them hard-coded within the method body here, could you maybe pull them out into a constant at the top? Something like:

```
class ZendeskService
  attr_accessor :ticket

  CUSTOM_FIELD_IDS = {
    contact_form: '360011490478',
    user_type: '360011798678',
    support_topics: '360011519218',
    telephone_number: '360011762698',
  }.freeze

(...)

  def send!
    ZendeskAPI::Request.create!(
      (...)
      custom_fields: [
        { id: CUSTOM_FIELD_IDS[:contact_form], value: 'contact_form' },
        { id: CUSTOM_FIELD_IDS[:user_type], value: ticket['user_type'] },
        { id: CUSTOM_FIELD_IDS[:support_topics], value: ticket['support_topics'] },
        { id: CUSTOM_FIELD_IDS[:telephone_number], value: ticket['telephone_number'] },
    ],
```